convert2rhel/backup/packages.py

```diff
@@ -73,6 +74,140 @@
 )
 _UBI_9_REPO_PATH = os.path.join(_RHSM_TMP_DIR, "ubi_9.repo")
 
+# Over time we want to replace this with pkghandler.RestorablePackageSet.
+class RestorablePackage(RestorableChange):
+    """Keep control of installed/removed RPM pkgs for backup/restore."""
+
+    def __init__(self, pkgs, reposdir=None, set_releasever=False, custom_releasever=None, varsdir=None):
+        super(RestorablePackage, self).__init__()
+
+        self.pkgs = pkgs
+        self.reposdir = reposdir
+        self.set_releasever = set_releasever
+        self.custom_releasever = custom_releasever
+        self.varsdir = varsdir
+
+        self._backedup_paths = []
+
+    def enable(self):
+        """Save version of RPM package.
+
+        :param reposdir: Custom repositories directory to be used in the backup.
+        :type reposdir: str
```

Comment:
I think this is some leftover, maybe it should be placed at the `__init__`?

Reply:
Yeah, thanks for catching that!